woocommerce/woocommerce-gateway-amazon-pay
# Fixed PHP notice on other admin pages.

### All Submissions:


### Changes proposed in this Pull Request:

Problem
* When going to administrative editing pages of other plugins a PHP notice appears, because the Amazon Payment Gateway expects global $post object for its meta widget, but there is none:
```
Notice: Trying to get property 'ID' of non-object
in /wp-content/plugins/woocommerce-gateway-amazon-payments-advanced/includes/class-wc-amazon-payments-advanced-order-admin.php
on line 153
```

Example
* `/wp-admin/admin.php?page=webappick-manage-feeds&action=edit-feed&feed=my-feed-id`

### How to test the changes in this Pull Request:

1. Install the plugin webappick-product-feed-for-woocommerce-pro.
2. Create a new product feed.
3. The PHP notice appears on the editing page of the product feed.

### Other information:


### Changelog entry

> Fixed PHP notice on admin pages of other plugins.
